docker/config/bootstrap.toml

```diff
@@ -0,0 +1,8 @@
+Port = 34001
+BootstrapOnly = true
+NotaryGroupConfig = "./notarygroup.toml"
+
```

Comment:
@tobowers regarding https://github.com/quorumcontrol/tupelo-integration-runner/pull/19#discussion_r349522322

Our other toml configs don't set it, but the ip is set manually on docker-compose so maybe it makes sense?